views/restaurants/index.ejs

```diff
@@ -1,37 +1,21 @@
 <%- contentFor('title') %>EcoFoodie
 <%- contentFor('body') %>
 <link rel="stylesheet" type="text/css" href="index.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> 
 
-<center><h1>EcoFoodie </h1></center>
-
-<a href="/restaurants/new/">Add New Restaurants </a><br>
-<h5>EcoFoodie restaurants near me </h5>
+<center><h1 class="restuarant_landing_title">EcoFoodie </h1></center>
 
+<a class = "addrestaurants"href="/restaurants/new/">Add New Restaurants </a><br>
+<h5 class="ecorestaurants_nearme" align="middle">EcoFoodie restaurants near me </h5>
 
 
 <% for(let record of records) { %> 
 <div class= "box">
-
-
-
-will continue edit this part  
-  <!-- <div class="media">
-    <img src="..." class="mr-3" alt="...">
-    <div class="media-body">
-      <h5 class="mt-0">Media heading</h5>
-      Cras sit amet nibh libero, in gravida nulla. Nulla vel metus scelerisque ante sollicitudin. Cras purus odio, vestibulum in vulputate at, tempus viverra turpis. Fusce condimentum nunc ac nisi vulputate fringilla. Donec lacinia congue felis in faucibus.
-    </div>
-  </div> -->
-
-
-
-
-
-    <span class="d-block p-2 bg-dark text-white">
+    <span class="d-block p-2 ">
         <div class="mousepointer">
-        
-            <img class="logo" src="images/chipotle_logo.jpg" alt="" align = "middle" hspace="15"><%= record.name %>
-        <div class = "address" align= "left" hspace="-100%"><%= record.address %></div>
+            <img class="restaurants_logo" src="images/chipotle_logo.jpg" alt="" align = "middle" hspace="12">
+            <span class =" boldtext"><%= record.name %> </span>
+        <div class = "address" align= "middle" hspace="-100%" ><%= record.address %></div>
  
 <div class= "rating"> 
```

Comment:
Same thing about indentation for this <div class="rating"> and the <div class="edit_restaurants"> below. Can you make sure that they are indented the proper amount?